### PR TITLE
NO-JIRA: Remove ILB from CNCC

### PR DESCRIPTION
**Pull request by martinkennelly**
CNCC is a cluster networked pod. Remove references to cloud provider internal LB.

### CODE REVIEW

**Comment by martinkennelly** [comment]
Note to self: deal with managed services use of a LB: `kubectl --kubeconfig $kc config set clusters.default.server "https://[${KUBERNETES_SERVICE_HOST}]:${KUBERNETES_SERVICE_PORT}"`

**Comment by martinkennelly** [comment]
Dont understand the managed service manifest version of CNCC. I assume it doesnt use platform ILB. Need to verify.

File: bindata/cloud-network-config-controller/managed/controller.yaml [hunk 1]
```diff
@@ -165,10 +165,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        - name: KUBERNETES_SERVICE_PORT
```
Comment (kyrtapz) [inline bindata/cloud-network-config-controller/managed/controller.yaml:168]: Note: `KUBERNETES_SERVICE_PORT` and `KUBERNETES_SERVICE_HOST` were not used in this container anyway.
CNCC uses a kubeconfig that is build here: https://github.com/openshift/cluster-network-operator/pull/2182/files#diff-2a7bd982b6883fe7152cf23cef357f7d9e2ee3c59d1cce19d1666d7939ae5c37L72-L90
The env vars used point to the local API service: https://github.com/openshift/cluster-network-operator/blob/1bb16ff35e90ad2e610fed12ea29e459c879c850/pkg/bootstrap/types.go#L123-L126

**Review by kyrtapz — APPROVED** [review]
The PR only changes how CNCC talks to the API in a standalone cluster since for HyperShift we are already using a local service.

/lgtm

**Comment by martinkennelly** [comment]
/retest

No failures pointing to a problem with this PR.

**Comment by martinkennelly** [comment]
/retest-required

Unrelated to PR

**Comment by martinkennelly** [comment]
/test e2e-vsphere-ovn-windows

"error waiting for Windows Machines to be provisioned"

**Comment by martinkennelly** [comment]
/retest

CI is borked

**Comment by martinkennelly** [comment]
/retest-required

CI broken..

**Pull request merged**